test/test_integration.py

```diff
@@ -82,6 +82,59 @@ def test_simple(self):
 """,
         )
 
+    def test_measurement(self):
+        myCircuit = Circuit.from_string(
+            """
+                version 3.0
+
+                qubit[3] qreg
+
+                Ry qreg[2], 2.34
+                Rz qreg[0], 1.5707963
+                Ry qreg[0], -0.2
+                CNOT qreg[1], qreg[0]
+                Rz qreg[0], 1.5789
+                CNOT qreg[1], qreg[0]
+                Rz qreg[1], 2.5707963
```

Comment:
This might be a bit overkill to only test the measure operation at the end.

We also want to test if the following measure operation works out correctly: 

```
measure qreg

//  should become

measure q[0]
measure q[1]
measure q[2]
```

Reply:
I have added the test but not awaiting 0.6.3

Reply:
I still miss a test where, e.g. 

```
qubit[3] qreg

H qreg[0]
CNOT qreg[0], qreg[1]
Z qreg[2]

measure qreg
```

becomes (not considering decomposition):

```
qubit[3] qreg

H qreg[0]
CNOT qreg[0], qreg[1]
Z qreg[2]

measure qreg[0]
measure qreg[1]
measure qreg[2]
```

We will release libqasm 0.6.3 soon and then be able to merge this PR.

Reply:
I tested the output of three consecutive measurements with 0.6.3 i.e,

```
measure qreg[0]
measure qreg[1]
measure qreg[2]
```
and it has passed without error. 

Regarding, `measure qreg`, I think we create a separate issue for this